test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -217,6 +217,7 @@
 
   <test id="Create and Restore Criu Checkpoint Image once - CheckpointDeadlock">
     <command>bash $SCRIPPATH$ $TEST_RESROOT$ $JAVA_COMMAND$ "$JVM_OPTIONS$ --add-opens java.base/jdk.internal.misc=ALL-UNNAMED  --add-exports java.base/openj9.internal.criu=ALL-UNNAMED" $MAINCLASS_DEADLOCK_TEST$ CheckpointDeadlock 1</command>
+    <output type="success" caseSensitive="yes" regex="no">User requested Java dump using</output>
```

Comment:
should we also update the test below?

Reply:
Yeah, those tests do Java dump as well, updated.